src/signals/incident-management/containers/IncidentDetail/index.test.js

```diff
@@ -385,32 +408,52 @@ describe('<IncidentDetail />', () => {
 
     it('onRequestIncident', () => {
       mapDispatchToProps(dispatch).onRequestIncident(42);
-      expect(dispatch).toHaveBeenCalledWith({ type: REQUEST_INCIDENT, payload: 42 });
+      expect(dispatch).toHaveBeenCalledWith({
+        type: REQUEST_INCIDENT,
+        payload: 42,
+      });
     });
 
     it('onPatchIncident', () => {
       mapDispatchToProps(dispatch).onPatchIncident({ patch: 'foo' });
```

Comment:
At lot of these seem to be testing the action directly instead of the IncidentDetails component

Reply:
Very true; I made a ticket for the refactoring of the `IncidentDetail` component. We can pick it up then.